lib/active_merchant/billing/gateways/hps.rb

```diff
@@ -126,7 +126,8 @@ def commit_credit_sale(money, card_or_token, options)
       end
 
       def add_reference(xml, transaction_id)
-        xml.hps :GatewayTxnId, transaction_id
+        reference = transaction_id.to_s.include?(';') ? transaction_id.split(';').first : transaction_id
```

Comment:
A small thing, but I noticed that in [the external docs that are currently in progress](https://paper.dropbox.com/doc/Spreedly-Gateway-Integration-Submission-Guide--A6XiaoDgktOANJkEt0FFIu0BAg-xKEbPETXVm7lslzOVgkYx#:uid=844585903520059154756786&h2=Code-Expectations), David stated a preference for using `|` as the separator.

Reply:
good call out, thank you!